hazelcast/src/main/java/com/hazelcast/map/impl/operation/PartitionWideEntryWithPredicateOperationFactory.java

```diff
@@ -155,4 +156,19 @@ public void readData(ObjectDataInput in) throws IOException {
         }
         return partitionToKeys;
     }
+
+    @Override
+    public Operation createOperation() {
```

Comment:
It's for the deserialiser to create an operation and then populate its fields from the byte stream